lib/traject/macros/fgdc.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Macros
+  # Macros for extracting FGDC values from Nokogiri documents
+  module FGDC
+    NS = { fgdc: 'http://www.fgdc.gov/metadata/fgdc-std-001-1998.dtd' }.freeze
+
+    def normalize_type
+      extract_fgdc('/*/idinfo/citation/citeinfo/geoform', translation_map: 'types')
+    end
+
+    def harvard_uuid(record)
+      urls = record.xpath('/*/idinfo/citation/citeinfo/onlink', NS).map(&:text)
+      urls.first.split('CollName=')[1]
+    end
+
+    def extract_uuid(record, context)
+      institution = context.settings.fetch('agg_provider')
+      if institution.include?('Harvard')
+        harvard_uuid(record)
+      elsif institution.include?('MIT')
```

Comment:
These xpaths looks the same to me. Can it be one elsif clause with `institution.include?('MIT') || institution.include?('Tufts')`

Reply:
done, thanks for that catch.